tensorstore/kvstore/tsgrpc/tsgrpc_test.cc

```diff
@@ -77,16 +77,28 @@ class TsGrpcMockTest : public testing::Test {
     ON_CALL(mock(), List).WillByDefault(Return(grpc::Status::CANCELLED));
   }
 
-  tensorstore::KvStore OpenStore() {
+  tensorstore::KvStore OpenStore(uint32_t max_sent_part_bytes = 2 * 1024 * 1024) {
     return kvstore::Open({
                              {"driver", "tsgrpc_kvstore"},
                              {"address", mock_service_.server_address()},
+                             {"max_sent_part_bytes", max_sent_part_bytes},
                          })
         .value();
   }
 
   MockKvStoreService& mock() { return *mock_service_.service(); }
 
+  void ExpectWriteRequestsAre(grpc::ServerReader<WriteRequest>* req,
```

Comment:
Avoid using EXPECT in nested functions; instead consider adding a matcher.

https://google.github.io/googletest/reference/matchers.html#defining-matchers